### PR TITLE
implement background scheduler

### PR DESCRIPTION
**Pull request by kghamilton89**
This PR is related to: https://discord.com/channels/879548962464493619/1221782765528289300/1222978196966871212

### CODE REVIEW

File: prompt_translation/03_create_dashboard.ipynb [hunk 1]
```diff
@@ -28,12 +28,15 @@
     "- `SOURCE_WORKSPACE`: The workspace id of the source dataset (unless otherwise specified, this will be `admin`)\n",
```
Comment (davanstrien) [inline prompt_translation/03_create_dashboard.ipynb:28]: <p>Make sure to update the  looks like a unfinished sentence?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/data-is-better-together/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by davanstrien** [comment]
@kghamilton89 thanks so much for this, I just had one question about a sentence in the notebook, but otherwise think this is ready to merge :)

**Comment by ignacioct** [comment]
I've done a separate PR implementing all changes discussed @kghamilton89 here: https://github.com/huggingface/data-is-better-together/pull/46

**Comment by davanstrien** [comment]
@kghamilton89 since we merged #46 I'll close this :)

**Pull request closed without merging**